utils/api.py

```diff
@@ -0,0 +1,37 @@
+from data.schema import GetEntity, CreteEntityResponse, Entity
+from utils.htttp_methods import HttpMethods
+
+BASE_URL = "http://localhost:8081/"
+
+
+class EntityApi:
+    @staticmethod
+    def create_new_entity(body: Entity) -> tuple[CreteEntityResponse, int]:
+        post_url = BASE_URL + "api/create"
+        response_body = HttpMethods.post(post_url, body.model_dump())
+        return CreteEntityResponse(id=response_body.text), response_body.status_code
+
+    @staticmethod
+    def get_entity_by_id(entity_id: int) -> tuple[GetEntity, int]:
+        get_url = BASE_URL + f"api/get/{entity_id}"
+        response_body = HttpMethods.get(get_url)
+        return GetEntity(**response_body.json()), response_body.status_code
```

Comment:
Давай лучше будем возвращать `response` целиком, а уже в тесте, либо в отдельных функция получать то, что нам необходимо.
ps. Это относится ко всем методам класса.

```suggestion
class EntityApi:
    @staticmethod
    def create_new_entity(body: Entity) -> tuple[CreteEntityResponse, int]:
        post_url = BASE_URL + "api/create"
        return HttpMethods.post(post_url, body.model_dump())

    @staticmethod
    def get_entity_by_id(entity_id: int) -> tuple[GetEntity, int]:
        get_url = BASE_URL + f"api/get/{entity_id}"
        return HttpMethods.get(get_url)
```